src/ucp/core/ucp_ep.c

```diff
@@ -489,10 +489,10 @@ ucp_ep_create_api_conn_request(ucp_worker_h worker,
     ucp_ep_destroy_internal(ep);
 out:
     if (status == UCS_OK) {
-        status = uct_iface_accept(conn_request->listener->wiface.iface,
+        status = uct_iface_accept(conn_request->listener->wifaces[conn_request->wiface_idx].iface,
```

Comment:
before the 'if' add:
`wiface = conn_request->listener->wifaces[conn_request->wiface_idx].iface`
to reduce code duplication